packages/library-base/src/commonMain/kotlin/io/realm/Realm.kt

```diff
@@ -69,12 +72,33 @@ interface Realm : TypedRealm {
      */
     override fun <T : RealmObject> objects(clazz: KClass<T>): RealmResults<T>
 
+    /**
+     * Returns a [RealmQuery] matching the predicate represented by [query].
+     *
+     * A reified version of this method is also available as an extension function,
+     * `realm.query<YourClass>(...)`. Import `io.realm.query` to access it.
+     *
+     * The results yielded by the query reflect the state of the realm at invocation time, so the
```

Comment:
I would swap this section with the next as this section talks about Results, which isn't available until you call `find` or `asFlow`